src/components/pages/blueprints-and-devices/device-display/Device.jsx

```diff
@@ -33,31 +36,57 @@ const useStyles = makeStyles((theme) => ({
    }
 }));
 
+
+
 const Device = (props) => {
    const classes = useStyles();
-   return (
-      <Card className={classes.cardContainer}>
-         <Typography variant="overline" className={classes.textSubtitle}>
-            Device Name
-         </Typography>
-         <Box className={`${classes.header} ${classes.text}`}>
-            <Typography variant="h5">AeroSpec 9</Typography>
-            <Box>
-               <IconButton>
-                  <BinIcon />
-               </IconButton>
-               <Button className={classes.activeBtn}>
-                  Active
-               </Button>
+
+   const [expanded, setExpanded] = React.useState(false);
+   const [outlined, setOutlined] = React.useState(null);
+   const [removed, setRemove] = React.useState(false);
```

Comment:
condense these to `useState([value])`

Reply:
I have fixed all of them and pushed again. Do I need to create a new pull request? If so, how can I do that?

Reply:
Any changes you make will be updated in the pull request, so there's no need to make another one!